test/services/dumper/dumper-sequelize.test.js

```diff
@@ -9,7 +9,7 @@ const defaultValuesModel = require('../../../test-expected/sequelize/db-analysis
 
 const Dumper = require('../../../services/dumper');
 
-function getDumper() {
+function getDumper(options = {}) {
```

Comment:
[MISC] I can't find where you are using `getDumper` with some options in this contribution ?

Reply:
Fixed via: https://github.com/ForestAdmin/lumber/pull/471/commits/eab7fc01787e75ac3a4ed7c9c93d1610a0b007b0

It now makes sense to add options see: https://github.com/ForestAdmin/lumber/pull/471/commits/eab7fc01787e75ac3a4ed7c9c93d1610a0b007b0#diff-614379289d3345516dbe409537ed835fR114